test/test_determination.py

```diff
@@ -62,11 +62,11 @@ def test_cpp_file(self):
     def test_test_file(self):
         """Test files trigger themselves and dependent tests"""
         self.assertEqual(
-            self.determined_tests(["test/test_jit.py"]), ["test_jit_profiling", "test_jit"]
+            self.determined_tests(["test/test_jit.py"]), ["test_jit"]
```

Comment:
`test_jit_profiling` was only included to satisfy the test. After removing the star imports in `test/test_jit_profiling.py ` they are no longer picked up by this test and can be removed. cc @malfet